docs/BUILD.md

```diff
@@ -627,3 +629,16 @@ Last but not least, is this completely necessary?
 
 * `images/*.yml`: The need to update `images/*.yml` is understood, as these packages change a lot. Even so, the utilities might be better structured as a separate external utility that is run manually. Most of the time, you just build from a static `rootfs.yml` or `installer.yml`. When you want to update them, you run `update-images` (or similar) and it updates them. Finally, you commit when a good build is ready.
 * `pkg/*/Dockerfile`: The need to generate `Dockerfile` for many of the packages may mean too much cross-dependency, which can be brittle. It is possible that this is necessary, and there is no other way around it. However, we should explore how we can simplify dependencies.
+
+## How to update eve-alpine package
+
+Unlike a lot of other Linux distributions, Alpine Linux doesn't provide historical versions of all its packages. In fact, Alpine Linux reserves the right to update packages with security patches behind the scenes resulting in content of Alpine x.y.z repositories shifting slightly from time to time. This, obviously, goes against the principle of reproducible builds and makes our `lfedge/eve-alpine` cache serve a double function: not only it is used to speed up the build, but it also may end up being the only place on the Internet where a certain Alpine Linux package version x.y.z could be available from.
```

Comment:
Can we move this to an upper section? Someone coming to the "how-to" is past the "why" and just wants to get it done. Conversely, someone curious as to "why" may not care about actually doing it.

Reply:
I am actually not quite sure what you mean, but tell you what -- let me commit it as-is and you can send a PR with an actual proposed move -- since it'll be easier to understand it that way